cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -207,27 +142,85 @@
             border-left: @spine-width solid @spine-color;
         }
     } );
+
+    .respond-to-max( @bp-xs-max, {
+        padding-top: 0;
+        overflow: hidden;
+
+        > li {
+            overflow: hidden;
+            float: left;
+        }
+    } );
 }
 
-.org-chart_categories {
-    .link_item {
-        margin-bottom: 0;
+.org-chart_nav {
+
+    li {
         border-top: 1px solid @node-border-color;
+        margin-bottom: 0;
         background: @gray-5;
 
         &:last-child {
             border-bottom: 1px solid @node-border-color;
         }
     }
 
-    .link_text {
-       border: none;
+    .org-chart_nav-item {
```

Comment:
How is this different from the `li` above? Related, how is the same class on L:177 different from this one? Lastly, does it need to be nested here? By BEM naming it should already be a child of `org-chart` and shouldn't need any more specificity.